src/button/index.tsx

```diff
@@ -36,6 +36,7 @@ export interface ButtonProps {
   readonly onDoneAnimationEnd?: () => void
   readonly tabIndex?: string
   readonly disabled?: boolean
+  buttonRef?: (button: HTMLButtonElement) => void
```

Comment:
How is it related to textfield focus?

Reply:
There is a ⤫ to clear the field. And I don't want to lose the focus in the field when I click on it, so I need to know if what you click on to blur is this button. See: 
See https://github.com/blablacar/ui-library/pull/227/files/b76cd20dbfc692b88326228ff018030fe9632105#diff-c0fb58a964df8de2d8c9613483ec4cd4R152